env.sh

```diff
@@ -0,0 +1,5 @@
+# These are required for the app to execute
+export FLASK_APP=${FLASK_APP-"politi_hack.flask"}
+export POLITI_HACK_CONFIG_PATH=${POLITI_HACK_CONFIG_PATH-$(readlink -m ./config.json)}
```

Comment:
Doesn't seem like `readlink` has a `-m` flag on the OSX bash... Maybe we could omit it for now?